tfhe/benches/core_crypto/bench.rs

```diff
@@ -0,0 +1,309 @@
+    } else if Scalar::BITS == 32 {
+        (
+            LweDimension(778),
+            StandardDev(0.000003725679281679651),
+            DecompositionBaseLog(18),
+            DecompositionLevelCount(1),
+            GlweDimension(3),
+            PolynomialSize(512),
+            LweBskBunchingFactor(2),
+            ThreadCount(5),
+        )
+    } else {
+        unreachable!()
+    }
+}
+
+fn multi_bit_pbs<Scalar: UnsignedTorus + CastInto<usize> + CastFrom<usize> + Sync>(
+    c: &mut Criterion,
+) {
+    // DISCLAIMER: these toy example parameters are not guaranteed to be secure or yield correct
+    // computations
+    // Define parameters for LweBootstrapKey creation
+
+    let (
+        mut input_lwe_dimension,
+        lwe_modular_std_dev,
+        decomp_base_log,
+        decomp_level_count,
+        glwe_dimension,
+        polynomial_size,
+        bunching_factor,
+        thread_count,
+    ) = get_bench_params::<Scalar>();
+
+    while input_lwe_dimension.0 % bunching_factor.0 != 0 {
+        input_lwe_dimension = LweDimension(input_lwe_dimension.0 + 1);
+    }
+
+    // Create the PRNG
+    let mut seeder = new_seeder();
+    let seeder = seeder.as_mut();
+    let mut encryption_generator =
+        EncryptionRandomGenerator::<ActivatedRandomGenerator>::new(seeder.seed(), seeder);
+    let mut secret_generator =
+        SecretRandomGenerator::<ActivatedRandomGenerator>::new(seeder.seed());
+
+    // Create the LweSecretKey
+    let input_lwe_secret_key =
+        allocate_and_generate_new_binary_lwe_secret_key(input_lwe_dimension, &mut secret_generator);
+    let output_glwe_secret_key: GlweSecretKeyOwned<Scalar> =
+        allocate_and_generate_new_binary_glwe_secret_key(
+            glwe_dimension,
+            polynomial_size,
+            &mut secret_generator,
+        );
+    let output_lwe_secret_key = output_glwe_secret_key.into_lwe_secret_key();
+
+    let multi_bit_bsk = FourierLweMultiBitBootstrapKey::new(
+        input_lwe_dimension,
+        glwe_dimension.to_glwe_size(),
+        polynomial_size,
+        decomp_base_log,
+        decomp_level_count,
+        bunching_factor,
+    );
+
+    // Allocate a new LweCiphertext and encrypt our plaintext
+    let lwe_ciphertext_in = allocate_and_encrypt_new_lwe_ciphertext(
+        &input_lwe_secret_key,
+        Plaintext(Scalar::ZERO),
+        lwe_modular_std_dev,
+        &mut encryption_generator,
+    );
+
+    let accumulator =
+        GlweCiphertext::new(Scalar::ZERO, glwe_dimension.to_glwe_size(), polynomial_size);
+
+    // Allocate the LweCiphertext to store the result of the PBS
+    let mut out_pbs_ct = LweCiphertext::new(
+        Scalar::ZERO,
+        output_lwe_secret_key.lwe_dimension().to_lwe_size(),
+    );
+
+    let id = format!("Multi Bit PBS {}", Scalar::BITS);
+    #[allow(clippy::unit_arg)]
+    {
+        c.bench_function(&id, |b| {
+            b.iter(|| {
+                black_box(multi_bit_programmable_bootstrap_lwe_ciphertext(
```

Comment:
unsure, I think I just reproduced something we had elsewhere in the benchmarks

https://github.com/zama-ai/tfhe-rs/blob/85dc0f01648e0ca2e56a4d63f22a7e3691db5f0a/tfhe/benches/boolean/bench.rs#L26

Reply:
what do you suggest instead ? removing it entirely ?

Reply:
what i would suggest is giving the output container to `black_box`

Reply:
Not sure I follow

Reply:
i meant to do the call normally without the black_box, then after that call `black_box(&mut out_pbs_ct)`

Reply:
ok, why is this better/preferred ?

Reply:
the way black_box works is. it takes an input and tells the compiler it can't assume anything about what's going to be done with that input. in this case, we want to make sure that the operation actually happens, so we pass the output to black_box. passing the return value doesn't do anything since the return value is a unit type

Reply:
Ok thanks !